### DISCUSSION TITLE
Pixel doubling on macOS

### DISCUSSION THREAD

**Discussion by hjellinek** [Interlisp/medley#1801] [Q&A] [answered]
I'm running Medley under XQuartz on a MacBook Pro with a 3840 x 2160 external display.  I love the giant display, except the Medley window is so small that text is difficult to read.

Can anyone suggest a way to render Medley larger, using pixel-doubling or the like?  Can I use SDL on top of X somehow?

**Comment by nbriggs** [comment, marked as answer]
ldesdl (SDL) is happy to open medley in a native macOS window, and then you can use pixel doubling.

**Comment by hjellinek** [comment]
Perfect, thanks!  I built `ldesdl` (`./makeright sdl`), added "`-ps 2`" to my `medley` command, and the result is I can see Medley clearly now.

**Comment by masinter** [comment]
this should be in the documentation or at least in the software/faq
